nikola/image_processing.py

```diff
@@ -27,25 +27,23 @@
 """Process images."""
 
 import datetime
+import gzip
 import os
-import lxml
 import re
-import gzip
 
+import lxml
 import piexif
 
 from nikola import utils
 
-Image = None
 try:
     from PIL import ExifTags, Image  # NOQA
 except ImportError:
     try:
```

Comment:
We can probably get rid of the nested try-except, `import Image` has been deprecated for years and is not supported by anything reasonably modern.

Reply:
If this is the case I'm dropping the nested import.